quantum/nodes/ibm-quantum-system/ibm-quantum-system.js

```diff
@@ -0,0 +1,133 @@
+module.exports = function(RED) {
+  'use strict';
+
+  const util = require('util');
+  const snippets = require('../../snippets');
+  const shell = require('../../python').PythonShell;
+  const errors = require('../../errors');
+
```

Comment:
Ideally this block of code should be at the top of the file, not within the exported function.